lib/capistrano/tasks/bundler.cap

```diff
@@ -28,7 +28,9 @@ namespace :bundler do
           options = []
           options << "--gemfile #{fetch(:bundle_gemfile)}" if fetch(:bundle_gemfile)
           options << "--path #{fetch(:bundle_path)}" if fetch(:bundle_path)
-          unless test(:bundle, :check, *options)
+          if test(:bundle, :check, *options)
+            output.log("The Gemfile's dependencies are satisfied, skipping installation")
```

Comment:
I'm not sure about this, but looking at https://github.com/capistrano/rails/blob/master/lib/capistrano/tasks/migrations.rake#L11 I think `#info` is more correct than `output.log`. However, I'm not sure which is correct in which circumstance. I need to do a little more research later.

Reply:
Turns out, `#info` and `#log` are forwarded to `output.info` and `output.log` respectively. So I'd suggest switching this to call `log` directly, without `output`. Then I'd merge.

Reply:
The point with `info` makes sense to me. Should we prefer that?

Reply:
@NikolayRys Yes, there is consensus among the maintainers. 😄